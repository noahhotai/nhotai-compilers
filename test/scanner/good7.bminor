fdksdjhfksljdfh
dfsdf
"fdsdfdsfdsfs"

dsfsdfjsljdf
dsfsdfjsljdf
1.333
sfdsdf737347;
dsf;
dfdsfl;
fdsdf;
fice = 9;
float 
while{
    
}
