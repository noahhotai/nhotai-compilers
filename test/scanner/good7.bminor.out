IDENTIFIER     fdksdjhfksljdfh
IDENTIFIER     dfsdf
STRING LITERAL fdsdfdsfdsfs
IDENTIFIER     dsfsdfjsljdf
IDENTIFIER     dsfsdfjsljdf
FLOAT LITERAL  1.333000
IDENTIFIER     sfdsdf737347
SEMICOLON      ;
IDENTIFIER     dsf
SEMICOLON      ;
IDENTIFIER     dfdsfl
SEMICOLON      ;
IDENTIFIER     fdsdf
SEMICOLON      ;
IDENTIFIER     fice
ASSIGN         =
INT LITERAL    0
SEMICOLON      ;
FLOAT          float
WHILE          while
LEFT BRACE     {
RIGHT BRACE    }
